Mathlib/Data/Setoid/Basic.lean

```diff
@@ -196,6 +196,25 @@ theorem eq_top_iff {s : Setoid α} : s = (⊤ : Setoid α) ↔ ∀ x y : α, s.R
   simp only [Pi.top_apply, Prop.top_eq_true, forall_true_left]
 #align setoid.eq_top_iff Setoid.eq_top_iff
 
+lemma sInf_equiv {S : Set (Setoid α)} {x y : α} :
+    letI := sInf S
+  x ≈ y ↔ ∀ s ∈ S, s.Rel x y := Iff.rfl
```

Comment:
```suggestion
    x ≈ y ↔ ∀ s ∈ S, s.Rel x y := Iff.rfl
```
It looks a bit ugly (maybe it would be better with `Iff.rfl` on the next line) but I think strictly speaking these are our conventions.